web/elm/src/Dashboard/Grid.elm

```diff
@@ -278,7 +278,7 @@ computeFavoritesLayout params cards =
     in
     { cards = result.allCards |> List.filter (isVisible params)
     , headers = headers |> List.filter (isVisible params)
-    , height = result.totalHeight
+    , height = result.totalHeight - padding
```

Comment:
Rather than subtracting `padding`, maybe we should change this line:
https://github.com/concourse/concourse/blob/5530bdc6ee6461af68d6d98cf73b9c2f8f723b71/web/elm/src/Dashboard/Dashboard.elm#L1438
to
```
 , Views.Styles.separator 0 
```

and then get rid of `Dashboard.Grid.Constants.sectionSpacerHeight` entirely (including deleting https://github.com/concourse/concourse/blob/5530bdc6ee6461af68d6d98cf73b9c2f8f723b71/web/elm/src/Dashboard/Dashboard.elm#L1445). This is more consistent with how `computeLayout` works - the bottom padding is included in the `.dashboard-team-pipelines` div